proof/invariant-abstract/AARCH64/ArchVCPUDomain_AI.thy

```diff
@@ -0,0 +1,597 @@
+(*
+ * Copyright 2024, Proofcraft Pty Ltd
+ *
+ * SPDX-License-Identifier: GPL-2.0-only
+ *)
+
```

Comment:
given that this only exists on one architecture (unless I missed something), would be good to have a comment saying what this theory is for